src/applications/hca/components/LoggedOutIntroContent.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { toggleLoginModal as toggleLoginModalAction } from 'platform/site-wide/user-nav/actions';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+import HCASubwayMap from './HCASubwayMap';
+import HcaOMBInfo from './HcaOMBInfo';
```

Comment:
`HCA` or `Hca`?
<img src="https://cloud.githubusercontent.com/assets/136959/13371810/91070356-dcf4-11e5-80cd-87ffc40ab8e8.gif" alt="eye twitch" title="eye twitch" />

Reply:
I really want it to be HCA, but the linter yells at you if you try it. I think with OMB being all caps, it doesn't like HCA & OMB back to back.

Reply:
😭